src/blog/models.py

```diff
@@ -54,3 +56,15 @@ class Meta:
 
     def __str__(self):
         return self.body
+
+class BlogPageGalleryImage(Orderable):
+    page = ParentalKey(BlogPage, on_delete=models.CASCADE, related_name='gallery_images')
+    image = models.ForeignKey(
+        'wagtailimages.Image', on_delete=models.CASCADE, related_name='+'
+    )
```

Comment:
While this works, I have to click the _Add gallery images_ button every time I want to add a new image. I'm worried that we will get annoyed by this in the future. Perhaps it's not something to be addressed right now, but I can imagine us creating/adding an input that allows us to select multiple images at once.

![afbeelding](https://github.com/drikusroor/brazil-blog/assets/8208970/3164b70c-8581-4b20-8b77-874b468adaa7)

Reply:
I agree, adding them one by one can become quite annoying. I'll add a new issue for this